### DISCUSSION TITLE
What Is the Current Status of the Project?

### DISCUSSION THREAD

**Discussion by BWKC** [NVIDIA/open-gpu-kernel-modules#332] [Q&A] [answered]
Are the modules in usable condition? I don't know why this project makes the project status unclear.

**Comment by YusufKhan-gamedev** [comment]
Its good enough for regular use but its power management capabilities are lacking(compared to its proprietary version). It requires the nvidia proprietary userland to be useful as of late.

**Comment by mtijanic** [comment, marked as answer]
Hi there. From the [official blog post](https://developer.nvidia.com/blog/nvidia-releases-open-source-gpu-kernel-modules/):
> For data center GPUs in the NVIDIA Turing and NVIDIA Ampere architecture families, this code is production-ready.
 
[snip]

> In this open-source release, support for GeForce and Workstation GPUs is alpha-quality. 

The current codebase targets primarily Tesla GPUs and data center usecases, and the more consumer facing features will come in a future driver release. See also the [latest driver readme](https://us.download.nvidia.com/XFree86/Linux-x86_64/515.57/README/kernel_open.html).
